js/ui/grid_core/ui.grid_core.editing.js

```diff
@@ -2649,6 +2650,8 @@ const EditingController = modules.ViewController.inherit((function() {
 
         _beforeCancelEditData: function() { },
 
+        _afterCancelEditData: function() { },
+
```

Comment:
Why this method is needed? Maybe it is better to put this._internalState = []; into it?